@siphiuel I can't. When use `npm start` got:
```
MacBook-Pro:status-react curikovatm$ npm start
npm ERR! missing script: start

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/curikovatm/.npm/_logs/2018-10-25T15_10_45_324Z-debug.log
```
Log:
```
0 info it worked if it ends with ok
1 verbose cli [ '/usr/local/bin/node', '/usr/local/bin/npm', 'start' ]
2 info using npm@5.5.1
3 info using node@v8.11.2
4 verbose stack Error: missing script: start
4 verbose stack     at run (/usr/local/lib/node_modules/npm/lib/run-script.js:151:19)
4 verbose stack     at /usr/local/lib/node_modules/npm/lib/run-script.js:61:5
4 verbose stack     at /usr/local/lib/node_modules/npm/node_modules/read-package-json/read-json.js:115:5
4 verbose stack     at /usr/local/lib/node_modules/npm/node_modules/read-package-json/read-json.js:402:5
4 verbose stack     at handleExists (/usr/local/lib/node_modules/npm/node_modules/read-package-json/read-json.js:366:20)
4 verbose stack     at FSReqWrap.cb [as oncomplete] (fs.js:312:19)
5 verbose cwd /Users/curikovatm/status-desktop/react-native-desktop/react-native-cli/status-react
6 verbose Darwin 17.7.0
7 verbose argv "/usr/local/bin/node" "/usr/local/bin/npm" "start"
8 verbose node v8.11.2
9 verbose npm  v5.5.1
10 error missing script: start
11 verbose exit [ 1, true ]
```
What else can I do to start it?

--- a/src/status_im/node/core.cljs
+++ b/src/status_im/node/core.cljs
@@ -146,7 +146,10 @@
   (let [restart {:db (assoc db :node/restart? true :node/address address)}]
     (case status
       :started (stop cofx)
-      :starting restart
+      :starting (do
+                  (when utils.platform/desktop?
+                    (status/stop-node))
+                  restart)
       :stopping restart
       (start cofx address))))
 

